hosted/Dockerfile

```diff
@@ -1,40 +1,46 @@
-FROM ubuntu:focal
+FROM ubuntu:focal as build
 
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 ARG HAPPY_ENV
 ENV HAPPY_ENV=$HAPPY_ENV
+ENV STACK_NAME="roaga-somaexplorer"
```

Comment:
Did the infra team reach out to you about whether this variable is available at build time?

Reply:
It's not available directly. "You could add those to your task definition as environment variables, which could then be used by your docker-compose." But that's not any easier for a user of this system than just changing this file. It is a feature they are thinking about though and are looking for feedback on.